core/chains/evm/log/broadcaster.go

```diff
@@ -389,6 +389,14 @@ func (b *broadcaster) eventLoop(chRawLogs <-chan types.Log, chErr <-chan error)
 
 	b.logger.Debug("Starting the event loop")
 	for {
+		// Replay requests take priority.
+		select {
+		case blockNumber := <-b.replayChannel:
+			b.onReplayRequest(blockNumber)
+			return true, nil
+		default:
+		}
+
 		select {
 		case rawLog := <-chRawLogs:
```

Comment:
What happens if we get something on the replayChannel, but there is already a message waiting on the chRawLogs channel (from before the replay). Would that cause issues once the replay starts?